hack/e2e.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# This script assumes its running from a pod and does the following:
+# * Processes the template to setup a local cincy instance
+# * Prepares a graph using test data
+# * Ensures generated graph is valid
+
+# Debug information
+oc whoami
+oc project
+
+# Apply oc template
+oc process -f ../dist/openshift/cincinnati.yaml \
+  -p IMAGE=${CINCY_E2E_IMAGE:-pipeline} \
+  -p IMAGE_TAG=${CINCY_E2E_IMAGE:-deploy} \
```

Comment:
nit: `CINCY` -> `CINCINNATI`.  I like "Cincy" when I'm in a hurry, but would rather avoid abbreviations in this write (hopefully) once, read-many source.

Reply:
Fixed (also ensured `IMAGE_TAG` reads `CINCINNATI_E2E_IMAGE_TAG`)